src/sdp/SDP.ts

```diff
@@ -28,16 +28,6 @@ export default interface SDP {
    */
   hasCandidatesForAllMLines(): boolean;
 
-  /**
-   * Replaces group attribute `a=group:` line with `a=group:BUNDLE audio video`
-   */
-  withBundleAudioVideo(): SDP;
```

Comment:
Any chance we could remove `SDP` and rename `DefaultSDP` to `SDP`? given its not even injected anywhere?

Reply:
Feel like what we should do but probably in another log where we go and audit all classes and make them internal or etc. I dont want this to be bigger :))